cli/experimental/service/README.md

```diff
@@ -0,0 +1,15 @@
+# SLSA verifier as a service
+
+This document is WIP.
+
+Command to run the service locally:
+
+```bash
+$ docker build -f service/Dockerfile .
```

Comment:
Should this README be under `experimental` dir given the path to the dockerfile?

Reply:
I think it needs to be run from slsa-verifier head, with path `cli/experimental/service/Dockerfile`, otherwise go mod vendor won't work. Then the `cd service` and the `COPY` in the Dockerfile also needs to be changed.

Reply:
i Have not renamed the folder in the example, thanks for catching. I take it means the folder re-arrangement is OK..